### ISSUE TITLE
Unable to open Cypress after upgrading to 9.4.0

### ISSUE THREAD

**Issue by jmezzacappa** [cypress-io/cypress#19986]
### Current behavior

When attempting to run `yarn cypress open`, I get this:
```bash
env: node\r: No such file or directory
error Command failed with exit code 127.
```

### Test code to reproduce

`yarn cypress open`

### Cypress Version

9.4.0

### Other

- MacOS
- Yarn 1.22.15
- Node v16.13.0
- ZSH

**Comment by jmezzacappa** [comment]
After manually changing `node_modules/.bin/cypress` EOLs from CRLF to LF, I'm able to open Cypress.

**Comment by adeonir** [comment]
Same here

**Comment by AtofStryker** [comment]
Hello @jmezzacappa and @adeonir! We are currently aware of the issue and are working to resolve it. I will provide an update ASAP.

**Comment by muratkeremozcan** [comment]
If you need a repo to reproduce it, or fully test a fix:

Clone [lerna-react-ts-cypress](https://github.com/muratkeremozcan/lerna-react-ts-cypress.git). Install. Stay on repo root level.

```bash
yarn install / yarn i / yarn # at root level; all sub-level dependencies are taken care of
yarn start:a / start:b / start:c # start app-*
# on a different tab
yarn cy:open-a / cy:open-b / cy:open-c #  open cypress test runner at app level
# on a different tab
yarn cy:run-a / cy:run-b / cy:run-c # run cypress headlessly at app level
```

Upgrade to 9.4.0 and repeat.

The duplicate issue has been closed https://github.com/cypress-io/cypress/issues/19989.

**Comment by jennifer-shehane** [comment]
We know exactly what the issue is and are issuing a patch 9.4.1 release immediately. I'm really sorry for the inconvenience. We'll be addressing the root issue that caused this faulty release before our next release https://github.com/cypress-io/cypress/issues/19771

**Comment by flotwig** [comment]
This issue has been fixed in `cypress@9.4.1`, please upgrade from `9.4.0`.
